test_utils/test_utils.go

```diff
@@ -42,9 +42,9 @@ import (
 )
 
 type ParseCheckAndInterpretOptions struct {
-	Config             *interpreter.Config
-	CheckerConfig      *sema.Config
-	HandleCheckerError func(error)
+	ParseAndCheckOptions *ParseAndCheckOptions
```

Comment:
Like in `ParseCheckAndCompileOptions`, reuse `ParseAndCheckOptions` instead of only having the checker config.